### PR TITLE
Added the event page for gsoc seminar and github workshop

### PR DESCRIPTION
**Pull request by tibrewalpratik17**
Please critically review it. Though i did nothing much other than content writing still need your reviews.

### CODE REVIEW

**Review by americast — DISMISSED** [review]
Looks good! Thanx @tibrewalpratik17 !

**Comment by tibrewalpratik17** [comment]
@americast review the github workshop page too

**Comment by tibrewalpratik17** [comment]
ping @americast

**Review by pranitbauva1997 — CHANGES_REQUESTED** [review]
Good! But I think you haven't added it to the events/index.html which you ideally should in order to be accessible from the website.

File: events/github.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <div class="ui middle aligned stackable grid container">
+      <div class="row">
+        <div class="eight wide column">
+          <h1 class="ui header" style="font-size: 3rem">About the Workshop</h1>
+          <p style ="font-size: 1.2rem">The first step towards open source is learning the workflow of Github and knowing the appropriate Git commands. KOSS organises the Github workshop to help the first and second years have an in-depth knowledge of Git so that they can start contributing to the open source world efficiently. In the workshop, the coders have a hands-on experience using Github as they learn from the basics to the comprehensive part of Git. </p>
+         
+          <div class="ui huge primary button" style="margin-top:0px;" href="http://git/resources">Checkout the Resources <i class="right arrow icon"></i></div>
+        </div>
+        <div class="six wide right floated column">
+          <img src="assets/images/wireframe/white-image.png" class="ui large bordered rounded image">
```
Comment (pranitbauva1997) [inline events/github.html]: This isn't being rendered.

File: events/github.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <a class="item">Signup</a>
+</div>
+
+<!-- Page Contents -->
+<div class="pusher">
+  <div class="ui inverted vertical masthead center aligned segment">
+
+   
+    <div class="ui text container"><br>
+      <h1 class="ui inverted header" style="font-size: 4rem">
```
Comment (pranitbauva1997) [inline events/github.html:57]: Too much of unnecessary spaces over here. Let's move the actual content a little above if it is feasible.

File: events/gsocseminar.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    <div class="ui middle aligned stackable grid container">
+      <div class="row">
+        <div class="eight wide column">
+          <h1 class="ui header" style="font-size: 3rem">About the Seminar</h1>
+          <p style="font-size: 1.35rem">GSOC seminar is an interactive session organised by KOSS for the first year and second year students to brief them about the annual GSOC programme. Students who participated in GSOC as mentors or mentees share their experiences.<br><br>This seminar also helps in solving many queries and FAQs of the inquisitive coders. Also a live demo of using Git and selecting organisations is given by the seniors which helps in boosting the confidence of the young ones.</p>
+         
+           <div class="ui huge primary button" style="margin-top:40px;" href="http://www.gsocresources.com">Checkout the Resources<i class="right arrow icon"></i></div>
+        </div>
+        <div class="six wide right floated column">
+          <img src="assets/images/wireframe/white-image.png" class="ui large bordered rounded image">
```
Comment (pranitbauva1997) [inline events/gsocseminar.html]: Again not rendered. Plus again the same too much gap issue as above.

**Review by pranitbauva1997 — CHANGES_REQUESTED** [review]
Good! Some changes need to be incorporated before merging.

**Review by americast — APPROVED** [review]
Looks cool to me! Thanx @tibrewalpratik17 !

**Review by pranitbauva1997 — APPROVED** [review]
The existing problems with this PR (which can be merged, but issue needs to be created) are:
 * The GSoC seminar should not be in "Past Events"
 * Github workshop isn't present in the main events page.
 * The image isn't being rendered

**Pull request merged**